src/main/scala/is/hail/io/index/IndexReader.scala

```diff
@@ -44,14 +44,14 @@ class IndexReader(hConf: Configuration, path: String, cacheCapacity: Int = 8) ex
   private val region = new Region()
   private val rv = RegionValue(region)
 
-  var cacheHits = 0L
-  var cacheMisses = 0L
+  private var cacheHits = 0L
+  private var cacheMisses = 0L
```

Comment:
these can probably be `private[this]` which turns them into actual fields (otherwise scala secretly creates get and set methods). It cannot be `private[this]` fields cannot be read by other objects of the same Class whereas `private` can be.